app/poros/food_poro.rb

```diff
@@ -0,0 +1,10 @@
+class FoodPoro
+  attr_reader :upc, :description, :brand, :ingredients
+
+  def initialize(info)
+    @upc = info[:foods][:gtinUpc]
+    @description = info[:foods][:description]
+    @brand = info[:foods][:brandOwner]
+    @ingredients = info[:foods][:ingredients]
+  end
+end
```

Comment:
You can just call this `Food`. Since you aren't inheriting from ApplicationRecord, I can tell it's just a plain old ruby object. Calling it a Poro makes it a little redundant.